--- a/Makefile
+++ b/Makefile
@@ -37,26 +37,30 @@ onedriver-launcher: $(shell find ui/ cmd
 
 
 install: onedriver onedriver-launcher
-	cp onedriver /usr/bin/
-	cp onedriver-launcher /usr/bin/
-	mkdir -p /usr/share/icons/onedriver/
-	cp pkg/resources/onedriver.svg /usr/share/icons/onedriver/
-	cp pkg/resources/onedriver.png /usr/share/icons/onedriver/
-	cp pkg/resources/onedriver-128.png /usr/share/icons/onedriver/
-	cp pkg/resources/onedriver.desktop /usr/share/applications/
-	cp pkg/resources/onedriver@.service /etc/systemd/user/
-	gzip -c pkg/resources/onedriver.1 > /usr/share/man/man1/onedriver.1.gz
+	mkdir -p  $(DESTDIR)/usr/bin/
+	cp onedriver $(DESTDIR)/usr/bin/
+	cp onedriver-launcher $(DESTDIR)/usr/bin/
+	mkdir -p $(DESTDIR)/usr/share/icons/onedriver/
+	cp pkg/resources/onedriver.svg $(DESTDIR)/usr/share/icons/onedriver/
+	cp pkg/resources/onedriver.png $(DESTDIR)/usr/share/icons/onedriver/
+	cp pkg/resources/onedriver-128.png $(DESTDIR)/usr/share/icons/onedriver/
+	mkdir -p $(DESTDIR)/usr/share/applications/
+	cp pkg/resources/onedriver.desktop $(DESTDIR)/usr/share/applications/
+	mkdir -p $(DESTDIR)/usr/lib/systemd/user/
+	cp pkg/resources/onedriver@.service $(DESTDIR)/usr/lib/systemd/user/
+	mkdir -p $(DESTDIR)/usr/share/man/man1/
+	gzip -c pkg/resources/onedriver.1 > $(DESTDIR)/usr/share/man/man1/onedriver.1.gz
 	mandb
 
 
 uninstall:
 	rm -f \
-		/usr/bin/onedriver \
-		/usr/bin/onedriver-launcher \
-		/etc/systemd/user/onedriver@.service \
-		/usr/share/applications/onedriver.desktop \
-		/usr/share/man/man1/onedriver.1.gz
-	rm -rf /usr/share/icons/onedriver
+		$(DESTDIR)/usr/bin/onedriver \
+		$(DESTDIR)/usr/bin/onedriver-launcher \
+		$(DESTDIR)/etc/systemd/user/onedriver@.service \
+		$(DESTDIR)/usr/share/applications/onedriver.desktop \
+		$(DESTDIR)/usr/share/man/man1/onedriver.1.gz
+	rm -rf $(DESTDIR)/usr/share/icons/onedriver
 	mandb
 
 
